cli/bench/delete.go

```diff
@@ -0,0 +1,36 @@
+package bench
+
+import (
+	"fmt"
+	"math/rand"
+	"reflect"
+	"sync"
+
+	"github.com/FZambia/tarantool"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+type DeleteRespData struct {
+	key   string
+	value string
+}
+
+func DeleteRequestsLoop(ctx context.BenchCtx, tarantoolConnection *tarantool.Connection, results *Results, waitGroup *sync.WaitGroup, benchActiveFlag *bool) {
+	getRandomTupleCommand := fmt.Sprintf("box.space.%s.index.%s:random", benchSpaceName, benchSpacePrimaryIndexName)
+	for *benchActiveFlag == true {
+		getRandomTupleResponse, err := tarantoolConnection.Exec(tarantool.Call(getRandomTupleCommand, []interface{}{rand.Int()}))
+		if err == nil {
+			data := getRandomTupleResponse.Data
+			if len(data) > 0 {
+				key := reflect.ValueOf(data[0]).Index(0).Elem().String()
+				_, err := tarantoolConnection.Exec(
+					tarantool.Delete(
+						benchSpaceName,
+						benchSpacePrimaryIndexName,
+						[]interface{}{key}))
+				CountRequest(err, results)
+			}
+		}
+	}
```

Comment:
It looks scary. Try to split this code into functions.

Reply:
This time we can't resolve this situatuon with significant improvement in readability